src/objects/Allocation__c.object

```diff
@@ -116,6 +116,20 @@
         <trackTrending>false</trackTrending>
         <type>Lookup</type>
     </fields>
+    <fields>
+        <fullName>Payment__c</fullName>
+        <deleteConstraint>SetNull</deleteConstraint>
+        <description>Attributes allocations to a Payment. All settled Payments roll up to this Allocation&apos;s GAU.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>Attributes allocations to a Payment. All settled Payments roll up to this Allocation&apos;s GAU.</inlineHelpText>
+        <label>Payment</label>
+        <referenceTo>npe01__OppPayment__c</referenceTo>
+        <relationshipLabel>GAU Allocations</relationshipLabel>
+        <relationshipName>Allocations</relationshipName>
```

Comment:
@lparrott Not sure if this was reviewed or not, but if not - what about calling this "Payment Allocations"?

Reply:
Do you mean the Label, Name or both? In any case, we don't have strong opinions on it, so "Payment Allocations" works fine, I can update that.

Reply:
I was thinking of the `<relationshipLabel>`. That's what would show up on the Payment Page Layout (after the admin adds the related list to the layout). What do you think @juliesnow?

Reply:
I'm pretty unfamiliar w/ how these fields, labels, descriptions get edited, generated, and are made available to users, so I'm deferring to you, @force2b and @lparrott + @pedrodalcol. In fact, I thought the descriptions we've proposed were visible through the UI, but it sounds like that's not right. @force2b, can you pls educate me re. where they show up?
We're hoping for:
a) A way for existing and new orgs to see the descriptions we proposed in the quip doc we've all been reviewing, as they are applicable for Opportunity and Payment Allocations (the current descriptions are Opportunity Allocation-specific and could be confusing to users once they enable Payment Allocations)
b) If that can't be managed w/ an update, we need to provide a way for users to change them, which it now sounds like you're saying also isn't possible. If true, I defer to others on how to proceed. This will impact us, moving fwd. I'd love to have a separate conversation/be educated about what can be updated w/ a package update, what can be edited by users after update, and what is cast in stone. I'm afraid that there will be more in that last category that are going to prove challenging as we move fwd.
@annejackson - fyi

Reply:
Very helpful. Thanks, so much, for taking the time to explain this, @force2b!

Reply:
@force2b @juliesnow I think this conversation got confused with Rollup descriptions, here we're trying to determine what the field description for the Payment lookup field on Allocations says. Please advise.

Reply:
Not necessarily. Because the field label/helptext/description changes are directly related to the implementation of this new field I don't see any issues with including those changes in this PR. However, it you would prefer to hold all the text changes until the full Payment-Allocations PR is merged that's fine. I was under the impression that there was a desire to get those label changes into the managed package sooner than later.